src/Mod/Assembly/CommandSolveAssembly.py

```diff
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+# /****************************************************************************
+#                                                                           *
+#    Copyright (c) 2023 Ondsel <development@ondsel.com>                     *
+#                                                                           *
+#    This file is part of FreeCAD.                                          *
+#                                                                           *
+#    FreeCAD is free software: you can redistribute it and/or modify it     *
+#    under the terms of the GNU Lesser General Public License as            *
+#    published by the Free Software Foundation, either version 2.1 of the   *
+#    License, or (at your option) any later version.                        *
+#                                                                           *
+#    FreeCAD is distributed in the hope that it will be useful, but         *
+#    WITHOUT ANY WARRANTY; without even the implied warranty of             *
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU       *
+#    Lesser General Public License for more details.                        *
+#                                                                           *
+#    You should have received a copy of the GNU Lesser General Public       *
+#    License along with FreeCAD. If not, see                                *
+#    <https://www.gnu.org/licenses/>.                                       *
+#                                                                           *
+# ***************************************************************************/
```

Comment:
The uniformity of the ascii box characters isn't correct. See line 2 and 22

Same for https://github.com/FreeCAD/FreeCAD/pull/10764/files#diff-08d614cbb4fb3a9554bc57f0639d31c6a5a5fcc83621114dd06bb9d7600a9ff5L2-R22

Reply:
Same for https://github.com/FreeCAD/FreeCAD/pull/10764/files#diff-ec35d691aafdacda9b015fd00e530d861d07a967356d4c57fadea06448bfb4e0R2-R22